libs/platform/src/lib/components/menu/menu-item.component.html

```diff
@@ -1,3 +1,4 @@
 <span class="fd-menu__title">
     <ng-content></ng-content>
 </span>
+<span *ngIf="isTrigger"   class="fd-menu__addon-after--submenu fd-menu__addon-after"></span>
```

Comment:
Not a big deal but please reorder the classes, the modifier class should be after the "block" that it's modifying:
```
<span *ngIf="isTrigger" class="fd-menu__addon-after fd-menu__addon-after--submenu"></span>
```

Reply:
Modified the order of the class.